conans/client/graph/graph_manager.py

```diff
@@ -290,11 +304,11 @@ def _recurse_build_requires(self, graph, builder, check_updates,
         for node in graph.ordered_iterate(nodes_subset):
             # Virtual conanfiles doesn't have output, but conanfile.py and conanfile.txt do
             # FIXME: To be improved and build a explicit model for this
-            if node.recipe == RECIPE_VIRTUAL:
-                continue
+            #if node.recipe == RECIPE_VIRTUAL:
```

Comment:
?

Reply:
``Virtual`` recipes couldn't have ``build_requires`` so far, only regular ``requires``. This is opening the door for this possibility.